packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -83,6 +83,29 @@ export function buildPlugin({
 						}
 					},
 				},
+				// TODO: Replace this with a proper outdir solution for Plugins
+				// But for now, let's just mark all wasm/bin files as external
+				{
+					name: "Mark externals",
+					setup(pluginBuild) {
+						if (pluginBuild.initialOptions.outfile) {
+							const outdir = dirname(pluginBuild.initialOptions.outfile);
+
+							pluginBuild.onResolve(
+								{ filter: /.*\.(wasm|bin)/ },
```

Comment:
Do you want a $ on the end of this regex, just in case you get a path like: `project.bindings/foo`, which would be a false positive, right?

Reply:
+1